<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1">
     <widget class="QComboBox" name="syntaxBox">
      <item>
       <property name="text">
        <string>Regular expression</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Wildcard</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fixed string</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="senseFilter">
      <property name="text">
       <string>Case sensetive filter</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="3" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Model</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTreeView" name="Model"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="columnBox">
      <item>
       <property name="text">
        <string>Full Name</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Passport Data</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Phone Number</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="Remove">
      <property name="text">
       <string>REMOVE</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Filter Syntax:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Filter Column:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="Add">
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Filter Pattern:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QCheckBox" name="senseSorting">
      <property name="text">
       <string>Case sensetive sorting</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="filterPattern"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1090</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="New_file"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open File ...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As ...</string>
   </property>
  </action>
  <action name="New_file">
   <property name="text">
    <string>New File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
